lib/cnet/punt/punt_kernel.c

```diff
@@ -155,8 +155,14 @@ punt_kernel_node_init(const struct cne_graph *graph __cne_unused, struct cne_nod
     ctx->sock = socket(AF_INET, SOCK_RAW, IPPROTO_RAW);
     if (ctx->sock < 0)
         CNE_ERR_RET("Unable to open IPv4 RAW socket\n");
-    if (CNET_ENABLE_IP6 && (ctx->sock6 = socket(AF_INET6, SOCK_RAW, IPPROTO_RAW)) < 0)
+
+#if CNET_ENABLE_IP6
```

Comment:
CNET_ENABLE_IP6 is a define we can test at runtime we can do the following, correct?

We can add `ctx->sock6 = -1;` above the `if (CNET_ENABLE_IP6 && ...` line, this way we do not need to have compile time option.